### PR TITLE
Unify the GKE and AKS cluster installation instructions

### LINKED ISSUE #5591: [PoC] Refactor Kyma cluster installation docs
**Description**

Most of the steps in the Kyma cluster installation on GKE and AKS are the same. Check if it is possible to get rid of the duplicated content and write one instruction with toggles only in particular steps which actually differ.

Investigate if we can:
- unify the Kyma cluster installation guidelines for GKE, AKS, and Gardener
- add the instruction on how to install Kyma with your own domain as a step at the end of the cluster installation (patch)
- move GCP Marketplace installation to a separate doc

**Reasons**

Duplicated content is more difficult to maintain. We want to have more user-friendly docs.

### PR DESCRIPTION
**Pull request by mmitoraj**
**Description**

Changes proposed in this pull request:

- Made an attempt to unify the GKE and AKS cluster installation instructions

**Related issue(s)**
Resolves #5591

### CODE REVIEW

**Comment by mmitoraj** [comment]
https://github.com/kyma-project/kyma/issues/5591 - preview

File: docs/kyma/04-03-cluster-installation.md [hunk 1]
```diff
@@ -9,266 +9,210 @@ This installation guide explains how you can quickly deploy Kyma on a cluster wi
 
 If you need to use Helm and access Tiller, complete the [additional configuration](#installation-use-helm) after the installation.
 
-Choose the installation type and get started:
+Install Kyma on a [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/) (GKE) or [Azure Kubernetes Service](https://azure.microsoft.com/services/kubernetes-service/) (AKS) cluster, or on a Kubernetes cluster deployed through [Gardener](https://gardener.cloud/).
```
Comment (kazydek) [inline docs/kyma/04-03-cluster-installation.md]:
```suggestion
Install Kyma on a [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/) (GKE) cluster, an [Azure Kubernetes Service](https://azure.microsoft.com/services/kubernetes-service/) (AKS) cluster, or on a Kubernetes cluster deployed through [Gardener](https://gardener.cloud/).
```

Comment (mmitoraj) [reply docs/kyma/04-03-cluster-installation.md]: Changed to:
```
Follow these guidelines to install Kyma on a chosen hyperscaler.
```

File: docs/kyma/04-03-cluster-installation.md [hunk 2]
```diff
@@ -5,270 +5,203 @@ type: Installation
...
+  AKS
+  </summary>
 
-8. Access the cluster using the link and login details provided in the **Kyma info** section on the **Application details** page.
+- [Microsoft Azure](https://azure.microsoft.com) account
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.14.6 or higher
+- [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
 
->**TIP:** Watch [this](https://www.youtube.com/watch?v=hxVhQqI1B5A) video for a walkthrough of the installation process.
+>**NOTE:** Running Kyma on AKS requires three [`Standard_D4_v3` machines](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes-general#dv3-series-1). You create these machines when you complete the **Prepare the AKS cluster** step.
```
Comment (mszostok) [inline docs/kyma/04-03-cluster-installation.md]: **Prepare the AKS cluster** section was removed now we have **Prepare the cluster**  :)

File: docs/kyma/04-05-use-your-own-domain.md [hunk 3]
```diff
@@ -7,116 +7,197 @@ This guide explains how to deploy Kyma on a cluster using your own domain.
...
+
+- A domain for your [Azure Kubernetes Service](https://azure.microsoft.com/services/kubernetes-service/) (AKS) cluster
+- [Microsoft Azure](https://azure.microsoft.com)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.14.6 or higher
+- Tiller 2.10.0 or higher
+- [Docker](https://www.docker.com/)
+- A [Docker Hub](https://hub.docker.com/) account
+- [az](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
+
+>**NOTE:** Running Kyma on AKS requires three [`Standard_D4_v3` machines](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes-general#dv3-series-1). You create these machines when you complete the **Prepare the AKS cluster** step.
```
Comment (mszostok) [inline docs/kyma/04-05-use-your-own-domain.md]: the same thing about **Prepare the AKS cluster** as above

but in general, I **really** like the new layout, pico bello 👍

File: docs/kyma/04-03-marketplace-installation.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+---
+title: Install Kyma through the GCP Marketplace
+type: Installation
+---
+
+Follow these steps to quickly install Kyma through the GCP Marketpalce:
```
Comment (kazydek) [inline docs/kyma/04-03-marketplace-installation.md]:
```suggestion
Follow these steps to quickly install Kyma through the GCP Marketplace:
```

File: docs/kyma/04-05-use-your-own-domain.md [hunk 5]
```diff
@@ -1,122 +1,203 @@
...
-### Delegate the management of your domain to Google Cloud DNS
+Delegate the management of your domain to Google Cloud DNS.
 
-> **NOTE**: Google Cloud DNS setup has to be done only once per DNS zone
+>**NOTE**: Google Cloud DNS needs to be set up only once per a DNS zone.
 
 Follow these steps:
 
-1. Export the project name, domain name, and DNS zone name as environment variables. Run the commands listed below:
+1. Export the project name, the domain name, and the DNS zone name as environment variables. Run thesecommands:
```
Comment (kazydek) [inline docs/kyma/04-05-use-your-own-domain.md]:
```suggestion
1. Export the project name, the domain name, and the DNS zone name as environment variables. Run these commands:
```

File: docs/kyma/04-05-use-your-own-domain.md [hunk 6]
```diff
@@ -1,122 +1,203 @@
...
 
 2. Create a DNS-managed zone in your Google project. Run:
 
-    ```
+    ```bash
     gcloud dns --project=$GCP_PROJECT managed-zones create $DNS_ZONE --description= --dns-name=$DNS_NAME
     ```
 
-    Alternatively, create it through the GCP UI. Navigate go to **Network Services** in the **Network** section, click **Cloud DNS** and select **Create Zone**.
+    Alternatively, create the DNS-managed zone through the GCP UI. In the **Network** section navigate to **Network Services**, click **Cloud DNS** and select **Create Zone**.
```
Comment (kazydek) [inline docs/kyma/04-05-use-your-own-domain.md]:
```suggestion
    Alternatively, create the DNS-managed zone through the GCP UI. In the **Network** section navigate to **Network Services**, click **Cloud DNS**, and select **Create Zone**.
```

File: docs/kyma/04-05-use-your-own-domain.md [hunk 7]
```diff
@@ -316,60 +252,124 @@ Follow these steps:
...
+
+    ```bash
     az network dns record-set txt delete -n "_acme-challenge.$SUB_DOMAIN" -g $RS_GROUP -z $DNS_DOMAIN --yes
     az network dns record-set txt create -n "_acme-challenge.$SUB_DOMAIN" -g $RS_GROUP -z $DNS_DOMAIN --ttl 60 > /dev/null
     az network dns record-set txt add-record -n "_acme-challenge.$SUB_DOMAIN" -g $RS_GROUP -z $DNS_DOMAIN --value $TXT_VALUE
     ```
-5. Go back to the first console, wait about 2 minutes and press enter.
 
-6. Export the certificate and key as environment variables. Run these commands:
+6. Go back to the first console, wait about 2 minutes and press **Enter**.
```
Comment (kazydek) [inline docs/kyma/04-05-use-your-own-domain.md]:
```suggestion
6. Go back to the first console, wait for about 2 minutes and press **Enter**.
```

**Pull request merged**